code/modules/mob/mob_defines.dm

```diff
@@ -136,7 +136,7 @@
 	var/stunned = 0
 	var/weakened = 0
 	var/losebreath = 0 //Carbon
-	var/intent = null//Living
+	var/intent = null//Living -- Depreciated? (a_intent below is your help/disarm/grab/harm)
```

Comment:
not too sure about this comment

Reply:
that variable is used in 1 place in the code, in a place I think is meant to use var/a_intent, I'm happy to dump the change if you want but it could bear a looking-at.

Reply:
I suppose we can leave it in for new coders to look at.